drivers/aks/aks_driver.go

```diff
@@ -795,27 +945,58 @@ func (d *Driver) createOrUpdate(ctx context.Context, options *types.DriverOption
 			osDiskSizeGBPointer = to.Int32Ptr(int32(driverState.AgentOsdiskSizeGB))
 		}
 
-		agentStorageProfile := containerservice.ManagedDisks
-		if driverState.AgentStorageProfile != "" {
```

Comment:
`agentStorageProfile` is not supported any more in the creating AKS API. I think this is the reason why you removed it here. Maybe we should clean it up such as https://github.com/rancher/kontainer-engine/pull/200/files#diff-fdee2f189d794fc3e135129d93b4d373L238

Reply:
Resolved :+1: